gradio/chat_interface.py

```diff
@@ -248,6 +248,8 @@ def __init__(
             )
         self.flagging_options = flagging_options
         self.flagging_dir = flagging_dir
+        if isinstance(chatbot, Chatbot):
+            chatbot.unrender()
```

Comment:
why are we unrendering the chatbot?

Reply:
We first .unrender() if it it's already rendered in a larger Blocks, and then we re-render it later on because we should only render it in a specific place within the chatinterface. This fixes https://github.com/gradio-app/gradio/issues/10645